mockingbird-compiler/build.gradle.kts

```diff
@@ -41,6 +41,11 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach
     kotlinOptions.freeCompilerArgs += "-Xopt-in=kotlin.RequiresOptIn"
 }
 
+val sourcesJar by tasks.registering(Jar::class) {
```

Comment:
I'm most likely missing something here, but doesn't Kotlin generates source jars out of the box?